29_Arithmetic2Bench/main.cpp

```diff
@@ -362,84 +411,16 @@ class ArithmeticBenchApp final : public examples::SimpleWindowedApplication, pub
 		};
 		m_device->updateDescriptorSets(1u, dsWrites, 0u, nullptr);
 
-		const uint32_t elementCount = Output<>::ScanElementCount;
-		const auto MinSubgroupSize = m_physicalDevice->getLimits().minSubgroupSize;
+		const uint32_t elementCount = 1024*1024;
```

Comment:
why isn't this a constexpr in your Application class ? What keeps this in sync with output buffer size?